rust/macros/module.rs

```diff
@@ -300,6 +300,25 @@ impl ModuleInfo {
 
         info
     }
+
+    /// Parse ["First", "Second"] into Some(vec!["First", "Second"])
+    fn parse_authors(it: &mut token_stream::IntoIter) -> Vec<String> {
+        let mut authors: Vec<String> = vec![];
+        let group = expect_group(it);
+        assert_eq!(group.delimiter(), Delimiter::Bracket);
+        let mut stream = group.stream().into_iter();
+        loop {
+            if let Some(s) = try_string(&mut stream) {
+                authors.push(s);
+            }
+            if try_punct(&mut stream).is_none() {
```

Comment:
Looks like this would accept any punctuation between the authors. We probably just want `,`, right?

Reply:
Fixed, see new commit.